sixtyfps_compiler/passes/materialize_fake_properties.rs

```diff
@@ -71,13 +85,14 @@ fn must_initialize(elem: &Element, prop: &str) -> bool {
     }
 }
 
-fn maybe_materialize(
-    property_declarations: &mut BTreeMap<String, PropertyDeclaration>,
+/// Returns a type if the property needs to be materialized, false otherwise
```

Comment:
Nitpick: The docs say that the function returns a bool but it doesn’t do that anymore